SensorsAnalyticsSDK/src/main/java/com/sensorsdata/analytics/javasdk/SensorsAnalytics.java

```diff
@@ -1419,7 +1415,11 @@ private static ObjectMapper getJsonObjectMapper() {
     return jsonObjectMapper;
   }
 
-  private static final String SDK_VERSION = "3.1.17";
+  private static CloseableHttpClient getCloseableHttpClient() {
+    return HttpClients.custom().setUserAgent("SensorsAnalytics Java SDK " + SDK_VERSION).build();
+  }
```

Comment:
It's more idiomatic in Java to call factory methods like this `createX` rather than `getX`. `get` usually implies reading an existing value, not constructing something from scratch.

Could you refactor to `createCloseableHttpClient`, please?